src/main/java/name/valery1707/kaitai/KaitaiGenerator.java

```diff
@@ -75,10 +75,8 @@ public KaitaiGenerator overwrite(boolean overwrite) {
 		return this;
 	}
 
-	private ProcBuilder process(Log log) {
+	private ProcBuilder process(Logger log) {
```

Comment:
\[Checkstyle\] WARNING: 'method def modifier' has incorrect indentation level 8, expected level should be 2\.